### PR TITLE
your eval implementation doesn't even work

### PR DESCRIPTION
**Pull request by phantamanta44**
merge or i'm calling the discord bot police

### CODE REVIEW

File: bot.js [hunk 1]
```diff
@@ -36,16 +36,9 @@ Discord.on('message', function (message) {
 		return message.reply('Aleyküm selam.');
 	  }
   });
-
-Discord.on('message', function (message) {
```
Comment (tonkku107) [inline bot.js]: Please add this back

**Comment by phantamanta44** [comment]
done!!!!!!!!

**Comment by ohlookitsderpy** [comment]
same

**Comment by ohlookitsderpy** [comment]
seems legit

**Pull request merged**